### PR TITLE
accounts-db/report number of duplicated accounts found in shrinking

### PR DESCRIPTION
**Pull request by HaoranYi**
#### Problem

We want to know how many duplicated accounts when we shrink. 

#### Summary of Changes

Add a stat to report number of duplicated accounts in shrinking.

Fixes #

### CODE REVIEW

File: accounts-db/src/accounts_db.rs [hunk 1]
```diff
@@ -583,6 +583,7 @@ impl AccountFromStorage {
 pub struct GetUniqueAccountsResult {
     pub stored_accounts: Vec<AccountFromStorage>,
     pub capacity: u64,
+    pub num_dups: usize,
```
Comment (brooksprumo) [inline accounts-db/src/accounts_db.rs]: Can this be the same as the datapoint name?

```suggestion
    pub num_duplicated_accounts: usize,
```

File: accounts-db/src/accounts_db.rs [hunk 2]
```diff
@@ -2020,6 +2022,11 @@ impl ShrinkStats {
                     self.storage_read_elapsed.swap(0, Ordering::Relaxed) as i64,
                     i64
                 ),
+                (
+                    "num_duplicated_accounts",
+                    self.num_duplicated_accounts.swap(0, Ordering::Relaxed) as i64,
```
Comment (brooksprumo) [inline accounts-db/src/accounts_db.rs]: The conversion is not necessary here:

```suggestion
                    self.num_duplicated_accounts.swap(0, Ordering::Relaxed),
```

File: accounts-db/src/accounts_db.rs [hunk 3]
```diff
@@ -2119,6 +2126,13 @@ impl ShrinkAncientStats {
                     .swap(0, Ordering::Relaxed) as i64,
                 i64
             ),
+            (
+                "num_duplicated_accounts",
+                self.shrink_stats
+                    .num_duplicated_accounts
+                    .swap(0, Ordering::Relaxed) as i64,
```
Comment (brooksprumo) [inline accounts-db/src/accounts_db.rs]: Same thing here

```suggestion
                    .swap(0, Ordering::Relaxed),
```

**Review by brooksprumo — COMMENTED** [review]
looks good, just some nits

File: accounts-db/src/accounts_db.rs (hunk 1 above)
Comment (HaoranYi) [reply accounts-db/src/accounts_db.rs]: Done.

File: accounts-db/src/accounts_db.rs (hunk 2 above)
Comment (HaoranYi) [reply accounts-db/src/accounts_db.rs]: Done.

File: accounts-db/src/accounts_db.rs (hunk 3 above)
Comment (HaoranYi) [reply accounts-db/src/accounts_db.rs]: Done.

**Review by jeffwashington — APPROVED** [review]
lgtm

**Review by brooksprumo — APPROVED** [review]
:shipit:

**Pull request merged**